### DISCUSSION TITLE
Laravel sanctum 3 blocks installation

### DISCUSSION THREAD

**Discussion by LarsAsDev** [opcodesio/log-viewer#147] [Q&A] [answered]
Is there a reason that Sanctum 3 is not supported by log-viewer?
I get the following message during installation:
```
Root composer.json requires laravel/sanctum ^3.0, found laravel/sanctum[v3.0.0, v3.0.1, 3.x-dev] but the package is fixed to v2.15.1 (lock file version) by a partial update and that version does not match. 
Make sure you list it as an argument for the update command.
```

Is there any way I can still install this package without downgrading to Sanctum v2.15.1?

**Comment by arukompas** [comment, marked as answer]
Hey @LarsAsDev , are you sure this is because of the LogViewer installation? Neither Log Viewer nor Livewire (its dependency) does not require Laravel Sanctum to exist.

Do you still get the same error when running `composer install` or `composer update`?

**Comment by LarsAsDev** [comment]
Running `composer update` was indeed the solution.

I'm upgrading from laravel 8 to 9, and apparently I'm not seeing everything correctly anymore...
Thanks for your help.
